Engine/ResourceManagement/Resources/Mesh.cpp

```diff
@@ -25,6 +25,10 @@ GLuint Mesh::GetVAO() const
 	return vao;
 }
 
+GLuint Mesh::GetEBO() const {
```

Comment:
code style

Reply:
but sometines ebo, vao, vbo is declared in capital leters, this way is more clear that you are geting the ebo rather than GetEbo i think :/